src/components/RadioButtonGroup/index.js

```diff
@@ -45,8 +61,62 @@ class RadioButtonGroup extends Component {
         return undefined;
     }
 
+    getCheckedOptionRef() {
+        const { value, options } = this.props;
+        const currentOptionIndex = options.findIndex(option => isOptionChecked(option, value));
+        return currentOptionIndex !== -1 ? this.optionsRefs[currentOptionIndex] : null;
+    }
+
+    generateRefsForOptions() {
+        const { options } = this.props;
+        return options.reduce(list => [...list, React.createRef()], []);
+    }
+
+    addRefsToOptions(options) {
+        return options.reduce(
+            (list, option, index) => [
+                ...list,
+                {
+                    optionRef: this.optionsRefs[index],
+                    ...option,
+                },
+            ],
+            [],
+        );
+    }
+
+    isMarkerActive() {
+        const { value, options } = this.props;
+        const reducer = (accumulator, option) => accumulator || isOptionChecked(option, value);
+
+        return options.reduce(reducer, false);
```

Comment:
what decide if the marker is visible is if there is a valid value passed so you can:
```
isMarkerActive() {
        const { value, options } = this.props;
        return options.some(option => option.value === value);
    }
```

Reply:
!option.disabled validation is also required cos a radiobuttongroup where all items are disabled should not display marker